gridworld/data/adapter/adapter.py

```diff
@@ -78,28 +61,28 @@ def save_session(self, session, save_path=None):
 
     def load_session(self, session_name, load_path=None):
         if load_path is None:
-            path = pathlib.Path(DATA_PREFIX).parent
+            path = pathlib.Path(self.dataset.get_data_path()).parent
             session_path = path / 'buffer' / session_name
         else:
             session_path = pathlib.Path(load_path)
         with open(session_path, 'rb') as f:
             compressed_session = f.read()
         return pickle.loads(bz2.decompress(compressed_session))
-    
-    def render_session_video(self, session, 
-            visualize=False, postprocess=True, 
-            render_size=(64, 64), outpath=None, 
+
+    def render_session_video(self, session,
+            visualize=False, postprocess=True,
+            render_size=(64, 64), outpath=None,
             single_turn=False
         ):
         """
 
         Args:
-            session (GameSession): 
+            session (GameSession):
             visualize (bool, optional): whether to render a high-resolution human friendly visualization
             debug (bool, optional): whether to add a debug info on each screen. Defaults to True.
             postprocess (bool, optional): whether to re-encode video using h264 codec. !!! REQUIRES THE LATEST FFMPEG !!!
         """
-        path = pathlib.Path(outpath or DATA_PREFIX)
+        path = pathlib.Path(outpath or self.dataset.get_data_path())
```

Comment:
I need to fix this bc of the recently updated `get_data_path` method

Reply:
Right this PR was based on https://github.com/iglu-contest/gridworld/pull/20

Reply:
I'll drop 814da99 so it's easier to merge.